src/tests/ftest/util/dmg_utils.py

```diff
@@ -371,6 +371,22 @@ def storage_query_device_health(self, uuid):
         """
         return self._get_json_result(("storage", "query", "device-health"), uuid=uuid)
 
+    def storage_query_target_health(self, rank, tgtid):
```

Comment:
It seems this function is not used. Is this for future work?

Reply:
Not sure. It was added for some reason. I removed it.